frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1902,6 +1902,7 @@ end
 
 function ReaderFooter:getDataFromStatistics(title, pages)
     local sec = _("N/A")
+    -- This is implemented by the Statistics plugin
```

Comment:
If a comment along these lines is useful, it should be above over on line 1899. That being said, to me the function above strongly implies it's there in order to be overridden by an actual implementation. :-)

Reply:
I will move it up to 1899. When I first started working through how to get the timing data, I got confused as to how this function was working. Having this comment might save someone else a bit of time finding where the override is, or at least, it would have saved me a bit of time.